README.md

```diff
@@ -157,6 +157,7 @@ The following input parameters are also supported for the Endor Labs GitHub acti
 | `scan_secrets` | Scan source code repository and generate findings for secrets. See also `scan_git_logs`. (Default: `false`) |
 | `scan_summary_output_type` | Set the desired output format to `table`, `json`, `yaml`, or `summary`. (Default: `json`) |
 | `scan_tools` | Scan source code repository for CI/CD tools. (Default: `false`) |
+| `scan_github_actions` | Scan source code repository for github actions used in workflow files to analyze vulnerabilities and malware. (Default: `false`) |
```

Comment:
nit: it may be better to have the inputs here and in action.yml sorted alphanumerically - moving  `scan_github_actions` to just after `scan_git_logs`